src/components/Footer/index.tsx

```diff
@@ -0,0 +1,102 @@
+'use client';
+
+import Image from 'next/image';
+import Link from 'next/link';
+import { P, H3 } from '@/styles/text';
+import { IconType } from '@/lib/icons';
+import { Flex } from '@/styles/containers';
+import FooterImage from '../../../public/images/footer-image.webp';
+import {
+  FooterContainer,
+  FooterSection,
+  FooterDiv,
+  IconDiv,
+  FooterLink,
+} from './styles';
+import Icon from '../Icon';
+
+type SocialLink = {
+  href: string;
+  type: IconType;
+};
+
+const socialLinks: SocialLink[] = [
+  {
+    href: 'https://www.facebook.com/ImmigrationJusticeProjectOfSanDiego/',
+    type: 'facebook',
+  },
+  { href: 'https://twitter.com/ijpsandiego?lang=en', type: 'x' },
+  {
+    href: 'https://www.linkedin.com/company/immigration-justice-project/',
+    type: 'linkedin',
+  },
+  { href: 'https://www.instagram.com/ijp.sandiego/', type: 'instagram' },
+];
+
+export default function Footer() {
+  return (
+    <FooterContainer>
+      <FooterDiv>
+        <FooterSection>
+          <Image
+            alt="footer image"
+            src={FooterImage.src}
+            placeholder="blur"
+            blurDataURL={FooterImage.src}
+            quality={100}
+            width={168}
+            height={66}
+          />
+          <hr style={{ width: '160px' }} />
+          <P $color="white">2727 Camino del Rio S</P>
+          <P $color="white">©2024 by Immigration Justice Project Pro Bono</P>
```

Comment:
let's update "Immigration Justice Project Pro Bono" to "ABA Immigration Justice Project"